src/ILVerify/src/VerificationResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection.Metadata;
+
+namespace ILVerify
+{
+    public class VerificationResult
+    {
+        public string TypeName { get; internal set; }
```

Comment:
We should not need TypeName. It can be fetched from MethodDefinitionHandle.

Reply:
Removed `TypeName`